### DISCUSSION TITLE
Pegel im HA auf der Karte anzeigen

### DISCUSSION THREAD

**Discussion by nraeth** [stephan192/hochwasserportal#53] [General]
Hallo zusammen,

gibt es eine Möglichkeit die Pegel (optimalerweise automatisch) korrekt auf der Landkarte im Home Assistant zu positionieren?

Die Geoposition der Pegel wird in den Portalen ja angegeben, soweit ich sehe aber nicht mit gescraped.

Aktuell scheitere ich aber sogar daran, die Location manuell hinzufügen. 

Gruß
Niels

**Comment by stephan192** [comment]
> gibt es eine Möglichkeit die Pegel (optimalerweise automatisch) korrekt auf der Landkarte im Home Assistant zu positionieren?

Ehrlich gesagt: Ich sehe keinen Mehrwert in der Anzeige. Was bringt es mir wenn ich die Position auf der Karte sehe?

> Die Geoposition der Pegel wird in den Portalen ja angegeben, soweit ich sehe aber nicht mit gescraped.

Ich würde nicht sagen, dass alle Portale die Position liefern. Zudem muss man sehr wahrscheinlich dann mehrere Seiten scrapen um zusätzlich zu den wichtigen Werten auch noch die Positionen zu bekommen.

> Aktuell scheitere ich aber sogar daran, die Location manuell hinzufügen.

Ich bin nicht der 100% Home Assistant Profi, aber wenn dann geht das Hinzufügen evtl. mittels Templatesensor.
